source/declarations.tex

```diff
@@ -1744,6 +1744,9 @@
 of the \grammarterm{parameter-declaration}{'s}
 \grammarterm{decl-specifier-seq}
 in a \grammarterm{template-parameter}\iref{temp.param}.
+The \tcode{auto} \grammarterm{type-specifier} can also be used
+as the \grammarterm{simple-type-specifier}
+in an explicit type conversion (functional notation)\iref{expr.type.conv}.
```

Comment:
It's true that [dcl.spec.auto.general]/6 is really weird, but it really just means

> A placeholder type P has the meaning given by the following.  If P appears in context X, it means&hellip;
>
> Otherwise, if P appears in context Y, it means&hellip;
>
> Otherwise, the program is ill formed.